elife/hostname.sls

```diff
@@ -15,7 +15,9 @@ set-hostname:
 
 set-hosts:
     host.present:
-        - ip: 127.0.0.1
+        - ip: 
+            - 127.0.0.1
+            - ::1
```

Comment:
state updated in Salt `2018.3`, now issues a warning if name is present for other IPs not in list

* https://docs.saltstack.com/en/latest/ref/states/all/salt.states.host.html#salt.states.host.present